src/concepts/glean.md

```diff
@@ -0,0 +1,72 @@
+
+**Basic product analytics are collected out-of-the-box in a standardized way.**
+A baseline of analysis is important for all our mobile application, from counting active users to retention and session times. This is supported out-of-the-box by the library and work consistently across our mobile products.
+
+**No custom code is required for adding new metrics to a product.**
+To make engineers more productive, the SDK keeps the amount of instrumentation code required for metrics as small as possible. Engineers only need to specify what they want to instrument, with which semantics and then record the data using the Glean SDK. The SDK takes care of storing & sending that data reliably.
+
+**Following [lean data practices](https://leandatapractices.com/) through SDK design choices.**
+It's easy to limit data to what's necessary and documentation can be generated easily, aiding both transparency & understanding for analysis.
+
+**Better data tooling integration due to standardized data types & registering them in machine-readable files.**
+When having collected data described in machine-readable files, our various data tools can read them and support metrics automatically, without manual work.
+
+**Due to common high-level concepts for metrics, APIs & data tools can better match the use-cases.**
+To make the choice easier for which metric type to use, we are introducing higher-level data types that offer clear and understandable semantics - when you want to count something, you use the _"count"_ type. This also gives us opportunities to offer better tooling for the data, both on the client and for data tooling.
+
+**Basic semantics on how the data is collected are clearly defined by the library.**
+To make it easier to understand the general semantics of our data, the Telemetry library will define and document when which kind of data will get sent. This gives data analysis common basic semantics.
+
+# How to use Glean
+
+*   [Integrate the Glean SDK / library](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/README.md) into your product.
+*   [File a data engineering bug](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data%20Platform%20and%20Tools&component=General&short_desc=Glean:%20Enable%20application%20id%20org.mozilla.myProduct) to enable your products application id.
+*   [Use Redash](https://sql.telemetry.mozilla.org/) to write SQL queries & build dashboards using your products datasets, e.g.:
+    *   `_org_mozilla_fenix_baseline_parquet`
+    *   `_org_mozilla_fenix_events_parquet`
+    *   `_org_mozilla_fenix_metrics_parquet`
+*   _(Work in progress)_ Use events and [Amplitude](https://sso.mozilla.com/amplitude) for product analytics.
+*   [Use Databricks](https://sso.mozilla.com/databricks) for deep-dive analysis.
+*   Use the Glean debug ping viewer for QA & development.
```

Comment:
Should we link [this document](https://docs.google.com/document/d/1LdXgXxOTDQQqD7_yGYj9TE80JQ3Wq2nx5vrgn7oDqt8/edit#) here?

Reply:
Good point!
We should probably move that document into the firefox-data-docs too once @akkomar isn't concerned about making the URL public anymore.